main.js

```diff
@@ -239,3 +254,10 @@ function view(viewName) {
     renderDetails.setAttribute('class', 'hidden')
   }
 }
+
+function buildCart(cartList) {
```

Comment:
Are you using this function for anything?

Reply:
I think I may have misunderstood this one. 
Define a function that takes a `cart` object and returns a DOM tree containing the count of items in the cart. Is that what this one was asking?

Reply:
More like:

```js
function buildCart(cart) {
  var $cart = document.createElement('span')
  $cart.classList.add('badge')
  $cart.textContent = 'Cart (' + cart.items.length + ')'
  return $cart
}
```

Looks like you were just updating the `textContent` of an item already in the page, so this function isn't needed.